model_analyzer/model_manager.py

```diff
@@ -127,6 +131,28 @@ def run_models(self, models):
             'ModelManager.model_variant_name_manager',
             model_variant_name_manager_dict)
 
+    def _create_ensemble_submodels(
+        self, model: ConfigModelProfileSpec
+    ) -> Optional[List[ConfigModelProfileSpec]]:
+        model_config_dict = ModelConfig.create_model_config_dict(
+            config=self._config,
+            client=self._client,
+            gpus=self._gpus,
+            model_repository=self._config.model_repository,
+            model_name=model.model_name())
+
+        model_config = ModelConfig.create_from_dictionary(model_config_dict)
+
+        if not model_config.is_ensemble():
+            return None
```

Comment:
should this return an empty list instead of None?

Reply:
Actually this function is dead code. I've removed it.